pandas/core/indexes/base.py

```diff
@@ -2183,7 +2183,7 @@ def _get_consensus_name(self, other):
                 return self._shallow_copy(name=name)
         return self
 
-    def union(self, other):
+    def union(self, other, sort=True):
```

Comment:
You need to implement this in subclasses of `Index`, eg. `DatetimeIndex`.
```bash
[pandas] find ./pandas -type f | grep .py$ |  xargs grep -n "def union"                                 17:41:36  ☁  fix-drop_duplicates-when-duplicate-column-names ☂ ✭
./pandas/core/common.py:353:def union(*seqs):
./pandas/core/dtypes/concat.py:214:def union_categoricals(to_union, sort_categories=False, ignore_order=False):
./pandas/core/indexes/base.py:2186:    def union(self, other):
./pandas/core/indexes/datetimes.py:983:    def union(self, other):
./pandas/core/indexes/datetimes.py:1036:    def union_many(self, others):
./pandas/core/indexes/multi.py:2584:    def union(self, other):
./pandas/core/indexes/range.py:416:    def union(self, other):
./pandas/core/indexes/timedeltas.py:496:    def union(self, other):
```

Reply:
@Giftlin : I don't think you have addressed this comment yet.

Reply:
Oh yeah.. will do it 👍